ui/apps/platform/src/Containers/Clusters/DelegateScanning/Components/DelegatedRegistriesTable.tsx

```diff
@@ -0,0 +1,300 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable no-param-reassign */
+/* eslint-disable react/no-array-index-key */
+// TODO: remove lint override after @typescript-eslint deps can be resolved to ^5.2.x
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+import { Button, Select, SelectOption, TextInput } from '@patternfly/react-core';
+import {
+    TableComposable,
+    Thead,
+    Tbody,
+    Tr,
+    Th,
+    Td,
+    TbodyProps,
+    TrProps,
```

Comment:
Items in order is easier on the eyes, especially after prettier wraps. Not a blocker to merge.

Reply:
I copied this straight out of the PF docs example, where I think they prefer a more semantic ordering. (I prefer alpha-ordering myself for unrelated imports from the core lib, but I see the appeal of this ordering for the Table imports.)

Reply:
unrelated, but fwiw the eslint options in ACSCS enforce alpha ordering for imports iirc